pandas/indexes/category.py

```diff
@@ -468,6 +468,24 @@ def take(self, indices, axis=0, allow_fill=True, fill_value=None):
                                            na_value=-1)
         return self._create_from_codes(taken)
 
+    def map(self, mapper):
+        """
```

Comment:
maybe in another PR can consolidate the doc-strings of ``map`` into base (when we combine them all there)